### PR TITLE
Update RomanNumberalsTest

### LINKED ISSUE #1293: Remove use of parameterized tests
We have some exercises that use parameterized tests. This makes it harder to do TDD since you can't ignore tests. We should convert these test suites to use individual tests instead of one parameterized test.

The exercises that need to be changed are:
- [x] raindrops
- [x] matrix
- [x] binary
- [ ] octal
- [x] scrabble-score
- [x] roman-numerals
- [x] prime-factors
- [x] atbash-cipher
- [x] pig-latin

If you want to work on this issue please only fix one exercise per PR 🙂 And please don't put "fixes #1293" in the PR description (unless you're fixing the last exercise that needs to be changed) as [that will automatically close this issue when the PR is merged](https://github.com/blog/1506-closing-issues-via-pull-requests)! 🙂

### PR DESCRIPTION
**Pull request by LuLechuan**
Fixes part of #1293 
For exercise: Roman Numbers 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/roman-numerals/src/test/java/RomanNumeralsTest.java [hunk 1]
```diff
@@ -1,54 +1,79 @@
...
-                {575, "DLXXV"},
-                {911, "CMXI"},
-                {1024, "MXXIV"},
-                {3000, "MMM"}
-        });
+    private RomanNumeral romanNumeral;
+
+    @Test
+    public void testSingleDigitArabicToRomanNumber() {
+        romanNumeral = new RomanNumeral(1);
```
[the remark this answers is not in the document]
Comment (LuLechuan) [reply exercises/roman-numerals/src/test/java/RomanNumeralsTest.java]: Yeah for this one I get your point. I will change as requested.

**Comment by LuLechuan** [comment]
Hi, @sjwarner-bp , for some larger inputs, do I need to provide comments about how to reach the solutions? For example, for 49, explaining that 49 is not 40 + 5 + 4 but rather 50 - 10 + 10 - 1, hence it leads to XLIX?

**Comment by sjwarner-bp** [comment]
@LuLechuan I wouldn't worry too much about that - hopefully the README explains this adequately and provides a useful source so users will be able to do some wider reading if they're struggling!

Test suites are normally just this implementation :slightly_smiling_face:

**Comment by LuLechuan** [comment]
@sjwarner-bp Ok! Thanks!

**Comment by sjwarner-bp** [comment]
@LuLechuan No problem :slightly_smiling_face:

**Pull request merged**